tests/servers/good.rb

```diff
@@ -333,5 +333,6 @@ def split_header_value(str)
 
 EM.run do
   EM.start_server("127.0.0.1", 9292, GoodServer)
+  EM.start_server("::1", 9293, GoodServer)
```

Comment:
I remember EventMachine being a little odd on jruby, but this may be fixed by now. And that error @jrafanie looks more like a potential issue in jruby. It is calling directly into java code.

Many places referencing jruby/jruby#775 when `::1` causes a jruby connection error.